src/components/ProfileList/ModerationProfileListItem.tsx

```diff
@@ -101,7 +109,7 @@ export const ModerationProfileListItem: React.FC<{ profile: ProfileModel }> = ({
       <div
         className={styles.container}
         onClick={() =>
-          router.push(`${AppRoutes.userProfile}/${profile.userId}`)
+          router.push(`${AppRoutes.dashboardProfile}/${profile.userId}`)
```

Comment:
I think we should add the onClick handler to the parent with the 'moderationFrame' class. This way, the entire tile in the UI is clickable, not just half of it.